dask/array/tests/test_reductions.py

```diff
@@ -471,32 +471,35 @@ def test_topk_argtopk1(npfunc, daskfunc, split_every):
     a = da.random.random(800, chunks=((120, 80, 100, 200, 300), ))
     b = da.random.random((10, 20, 30), chunks=(4, 8, 8))
 
+    npa = a.compute()
+    npb = b.compute()
```

Comment:
We can certainly do this. My temptation would be to write something like this. Though I leave it up to you :)

```python
    npa = np.random.random(800)
    npb = np.random.random((10, 20, 30))

    a = da.from_array(npa, chunks=((120, 80, 100, 200, 300), ))
    b = da.from_array(npb, chunks=(4, 8, 8))
```